### ISSUE TITLE
WhateverGreen blanks out the screen on Ventura + Radeon 6600 XT

### ISSUE THREAD

**Issue by Ivorforce** [acidanthera/bugtracker#2276]
Here's my config (WhateverGreen intentionally disabled): https://gist.github.com/Ivorforce/bb84b09d35fe5f7168d84795afab77c0

Full hardware info:

- Mainboard: [Gigabyte Z590i Vision D](https://www.gigabyte.com/Motherboard/Z590I-VISION-D-rev-10/sp#sp)
- CPU: [Intel i7-11700k](https://www.intel.com/content/www/us/en/products/sku/212047/intel-core-i711700k-processor-16m-cache-up-to-5-00-ghz/specifications.html)
- GPU: [Radeon 6600 XT](https://www.amd.com/en/products/graphics/amd-radeon-rx-6600-xt)

Software:
- macOS Ventura (13.3.1 (a))
- WhateverGreen-1.6.4
- OpenCore-0.9.1-DEBUG

IIRC the install worked fine with WhateverGreen, but since then I had to disable it. When enabled, everything boots as normal, but when macOS should appear, I get no video output on either HDMI or thunderbolt.

I have no logs. If you need any, let me know what to do and how to provide them to you. Cheers!

**Comment by lin72h** [comment]
I have the same problem, just replug the display connector i can see the log screen, hope someone fix it

**Comment by vit9696** [comment]
Try with different `agdpmod` settings. This usually helps with 5xxx and 6xxx GPUs.

**Comment by Ivorforce** [comment]
Ok, I found a fix!

I cleaned up my config.plist DeviceProperties -> Add, and added one entry regarding graphics I found from [this repo](https://github.com/SchmockLord/Gigabyte-Z590i-Vision-D-11900k):

```xml
<dict>
<key>PciRoot(0x0)/Pci(0x1,0x0)/Pci(0x0,0x0)/Pci(0x0,0x0)/Pci(0x0,0x0)</key>
<dict>
<key>device-id</key>
<data>v3MAAA==</data>
<key>shikigva</key>
<integer>128</integer>
</dict>
```

Now, I was able to activate WhateverGreen. agdpmod remains unchanged (`agdpmod=pikera`). I've uploaded my new [config.plist](https://gist.github.com/Ivorforce/91b64113a3a9c7dc344565ec13e5ab0c) for anyone that comes across this. Cheers!
